x-pack/legacy/plugins/canvas/.storybook/storyshots.test.js

```diff
@@ -12,6 +12,11 @@ import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-story
 import styleSheetSerializer from 'jest-styled-components/src/styleSheetSerializer';
 import { addSerializer } from 'jest-specific-snapshot';
 
+// Several of the renderers, used by the runtime, use jQuery.
+import jquery from 'jquery';
+global.$ = jquery;
```

Comment:
Does it really need to be global? That's going to cause issues for pages that embed this script. If they do require jquery we could probably rewrite those global accesses with something like the webpack define plugin.

Reply:
The `plot` and `pie` renderers use `flot`, which uses `jQuery`.  This is in the storyshots test, though, so no one is going to embed this script.